test/test_utils.rb

```diff
@@ -1,6 +1,21 @@
 require 'helper'
 
 class TestUtils < JekyllUnitTest
+  context "The \`Utils.deep_merge_hashes\` method" do
```

Comment:
I'll digress that test to @jekyll/core, but the convention is always (if you test properly):

```ruby
describe ClassName do
  context "#method" do
    # Tests
  end
end
```

However, we do not fully utilize the convention of what we are using, we go between Minitest Spec and Minitest Unit in ways that prevent inferring context properly, which should change here soon... hopefully.

This test however is enforcing a context driven testing environment.